service/src/integrationTest/java/at/jku/isse/ecco/service/test/GeneralTest.java

```diff
@@ -19,10 +19,12 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-public class Test {
+import org.testng.annotations.Test;
 
+public class GeneralTest {
```

Comment:
There are no "General" tests, write tests with intention and put them in an aptly named file

Reply:
done by you